src/wokwi_client/file_ops.py

```diff
@@ -22,3 +22,16 @@ async def upload_file(
 async def upload(transport: Transport, name: str, content: bytes) -> ResponseMessage:
     params = UploadParams(name=name, binary=base64.b64encode(content).decode())
     return await transport.request("file:upload", params.model_dump())
+
+
+async def download(transport: Transport, name: str) -> ResponseMessage:
+    return await transport.request("file:download", {"name": name})
+
+
+async def download_file(transport: Transport, name: str, local_path: Optional[Path] = None) -> None:
```

Comment:
WDYT about removing `download_file` and moving the logic to the API client? 

The idea is that most users won't use the `file_ops.py`, as these are the low level wrappers for the API calls. I don't have a strong opinion here - it also depends on what solution we end up using for the sync client (if we end up duplicating the WokwiClient code, then it's better to keep more logic in the file_ops to reduce the amount of duplicated code)

Reply:
I have found a way to create the wrapper based on an async client without duplicating the code. So I have moved the download_file logic to the API client.